mivs/site_sections/mivs_applications.py

```diff
@@ -168,6 +168,8 @@ def confirm(self, session, csrf_token=None, decision=None):
             raise HTTPRedirect('index?message={}', 'You did not have any games accepted')
         elif studio.group:
             raise HTTPRedirect('index?message={}', 'Your group has already been created')
+        elif studio.after_confirm_deadline:
```

Comment:
This fixes https://github.com/magfest/mivs/issues/36, though not in the way the issue suggested.

Basically there's no way to change a game's status outside of an action someone takes... unless we want to write a daemon that checks all the games' confirmation deadlines every once in a while. That seemed like overkill. As far as I could tell, we really needed was to make sure studios can't confirm their acceptance after their deadlines, which is taken care of by this redirect here.